File: fizzbuzz.cpp
// f6 to run in vs code
#include <iostream>
using namespace std;

int main() {
    for (int h = 1; h < 101; ++h) {
        if (h % 3 == 0) {
            cout << "Fizz";
        }
        if (h % 5 == 0) {
            cout << "Buzz";
        }
        if (h % 3 != 0 && h % 5 != 0) {
            cout << h;
        }
        cout << endl;
    }
    return 0;
}

File: fizzbuzz.java
class fizzbuzz {
    public static void main(String[] args) {
        String word;
        for (int t = 1; t < 101; ++t) {
            word = "";
            if (t % 3 == 0) 
                word += "Fizz";
            if (t % 5 == 0)
                word += "Buzz";
            if (word == "")
                System.out.println(t);
            else
                System.out.println(word);
        }
    }
}
